recycle-node.rb

```diff
@@ -155,10 +158,9 @@ def cmd_successful?(cmd)
   system cmd
 end
 
-def execute(cmd, can_fail: false)
+def execute(cmd)
   log cmd
-  result = `#{cmd}` # TODO: Use open3
-  raise "Command: #{cmd} failed." unless (can_fail || $?.success?)
+  result = `#{cmd}`
```

Comment:
you can just end this method with the ``#{cmd}`` line - you don't need to assign to `result` .